src/Ztm.Zcoin.NBitcoin.Tests/RpcTests.cs

```diff
@@ -17,7 +17,9 @@ public class RpcTests : IDisposable
         public RpcTests()
         {
             this.builder = NodeBuilderFactory.CreateNodeBuilder(GetType().Name);
-            this.node = this.builder.CreateNode(true);
+            this.node = this.builder.CreateNode(false);
+            this.node.ConfigParameters.Add("printtoconsole", "0");
```

Comment:
We should apply this in `NodeBuilderFactory.CreateNodeBuilder` instead so we don't need to do this everywhere.

Reply:
agree